docs/1.2/architecture/index.md

```diff
@@ -10,23 +9,58 @@ Sawtooth, an enterprise blockchain platform for building distributed
 ledger applications and networks.
 
 This guide starts by explaining the important concepts of
-`global state`{.interpreted-text role="term"} and
-`Sawtooth batches<batch>`{.interpreted-text role="term"}. Next, it
-describes key parts of the `validator`{.interpreted-text role="term"}
-and other core features, including the journal for block management,
-consensus, transaction scheduling, permissioning, and more.
-
-![](images/arch-sawtooth-overview.*){.align-center width="100.0%"}
-
-::: toctree
-architecture/global_state architecture/transactions_and_batches
-architecture/journal architecture/scheduling architecture/rest_api
-architecture/validator_network architecture/permissioning_requirement
-architecture/injecting_batches_block_validation_rules
-architecture/events_and_transactions_receipts PBFT Consensus
-\<<https://sawtooth.hyperledger.org/docs/pbft/releases/latest/architecture.html>\>
-PoET 1.0 Consensus \<architecture/poet>
-:::
+`global state` and `Sawtooth batches`. Next, it
+describes key parts of the `validator` and other core features,
+including the journal for block management, consensus, transaction
+scheduling, permissioning, and more.
+
+
+<img alt="Sawtooth architecture"
+src="/images/1.2/arch-sawtooth-overview.svg">
+
+
+* [Global State](global_state)
```

Comment:
This section should probably just be simplified to the top level docs (Global State, Transaction and Batches, etc) because the labels that are being pointed to in several places wont exist. Or they should be switch to not use the "-lable" and use the correct header instead. Also these should probably be liquid links e.i ({% link doc_path%})